### PR TITLE
Add Computer Theory Book in Spanish Books

### PR DESCRIPTION
**Pull request by Rguarnizo**
Adding Computer Theory Book of Rodrigo de Castro Korgi, Using in National University of Colombia

## What does this PR do?
Add resource, Add the book of an excellent Professor in the Area of the Computing Theory.

### Why is this valuable (or not)?
The Professor Rodrigo de Castro Korgi has investigated in the Study Area and is one of the great contributors to the area, and facilitates learning in many aspects.

### How do we know it's really free?
This book is available digitally in the repository of the National University of Colombia, each time a new version of the book comes out of the previous book is released, this is the previously published version and that is marketed, but the content is very Similar and pretty good.

### For book lists, is it a book? For course lists, is it a course? etc.
Yes its a book, using in National University of Colombia for study all Computer Theory aspects.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-es.md [hunk 1]
```diff
@@ -94,6 +94,7 @@
 * [Breves Notas sobre Autómatas y Lenguajes](https://lya.fciencias.unam.mx/jloa/publicaciones/automatasyLenguajes.pdf) (PDF)
 * [Breves Notas sobre Complejidad](https://lya.fciencias.unam.mx/jloa/publicaciones/complejidad.pdf) (PDF)
 * [Breves Notas sobre Teoría de la Computación](https://lya.fciencias.unam.mx/jloa/publicaciones/teoria.pdf) (PDF)
+* [Teoría de la computación (Lenguajes,autómatas,Gramaticas) | UNAL](http://ciencias.bogota.unal.edu.co/fileadmin/Facultad_de_Ciencias/Publicaciones/Archivos_Libros/Libros_Matematicas/_Teoria_de_la_Computacion___lenguajes__automatas__gramaticas._/teoriacomputacion.pdf) (PDF)
```
Comment (eshellman) [inline books/free-programming-books-es.md:97]: Don't invent titles! Use the title given by the resource. "UNAL" isn't in the title.
Please add author names where appropriate.

Comment (LuigiImVector) [inline books/free-programming-books-es.md:97]:
```suggestion
* [Teoría de la computación (Lenguajes,autómatas,Gramaticas) | UNAL](http://ciencias.bogota.unal.edu.co/fileadmin/Facultad_de_Ciencias/Publicaciones/Archivos_Libros/Libros_Matematicas/_Teoria_de_la_Computacion___lenguajes__automatas__gramaticas._/teoriacomputacion.pdf) - Rodrigo De Castro Korgi (PDF)
```
Added author's name

Comment (Rguarnizo) [reply books/free-programming-books-es.md]: Understood, the changes have already been published.

Comment (davorpa) [reply books/free-programming-books-es.md]: Buen uso de `:` para separar un subtítulo importante

**Review by davorpa — APPROVED** [review]

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**